pkgs/top-level/python-packages.nix

```diff
@@ -5063,6 +5063,26 @@ in {
     };
   };
 
+  python-jose = buildPythonPackage rec {
```

Comment:
We have a new policy for python to put python packages into a separate file and only reference them here:

See also:

```
$ chromium "file://$(nix-build --no-out-link doc)//share/doc/nixpkgs/manual.html#contributing-guidelines"
```